lms/static/scripts/frontend_apps/errors.js

```diff
@@ -52,3 +65,44 @@ export class APIError extends Error {
     this.details = data.details;
   }
 }
+
+/**
+ * Does the current error object represent an authorization error or need
+ * to re-authorize from the back end?
```

Comment:
This is perhaps a little wordy. Could perhaps be phrased as say, "Return true if an error should be treated as an authorization error."